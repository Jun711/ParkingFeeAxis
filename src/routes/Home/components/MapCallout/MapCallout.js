import React, { Component } from 'react';
import { Text } from 'react-native';

import styles from './MapCalloutStyles';

export default class MapCallout extends Component {

  constructor(props) {
    super(props);
    this.state = {...props};
  }

  render() {
    return (
      <Text style={styles.mapCalloutText}
            selectable={true}
            adjustsFontSizeToFit={true}
            allowFontScaling={true}
      >{this.props.text}</Text>
    )
  }
}
