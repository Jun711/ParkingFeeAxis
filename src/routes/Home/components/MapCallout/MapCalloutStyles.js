const styles = {
  mapCalloutText: {
    borderColor: "#00264d",
    borderWidth: 1,
    fontSize: 8,
    padding: 5,
    position: "relative",
    zIndex: 5,
  },
};

export default styles;