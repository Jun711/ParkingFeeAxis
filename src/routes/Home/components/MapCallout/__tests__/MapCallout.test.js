import 'react-native'
import React from 'react'
import MapCallout from '../MapCallout'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('<MapCallout />', () => {
  it('MapCallout renders correctly', () => {
    const mapCalloutTree = renderer.create(
      <MapCallout />
    ).toJSON()
    expect(mapCalloutTree).toMatchSnapshot()
  })
})

