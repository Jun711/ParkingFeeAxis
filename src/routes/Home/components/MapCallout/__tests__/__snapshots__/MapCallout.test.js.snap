// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapCallout /> MapCallout renders correctly 1`] = `
<Text
  accessible={true}
  adjustsFontSizeToFit={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  selectable={true}
  style={
    Object {
      "backgroundColor": "#fff",
      "borderColor": "#000",
      "borderWidth": 1,
      "fontSize": 10,
      "padding": 5,
      "position": "relative",
      "zIndex": 5,
    }
  }
/>
`;

exports[`MapCallout renders correctly 1`] = `
<Text
  accessible={true}
  adjustsFontSizeToFit={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  selectable={true}
  style={
    Object {
      "backgroundColor": "#fff",
      "borderColor": "#000",
      "borderWidth": 1,
      "fontSize": 10,
      "padding": 5,
      "position": "relative",
      "zIndex": 5,
    }
  }
/>
`;
